### PR TITLE
fixing range proofs bound check

### PR DESCRIPTION
**Pull request by maycon-mello**
main

### CODE REVIEW

File: packages/wasm/src/services/credential/bound-check.ts [hunk 1]
```diff
@@ -61,60 +62,38 @@ export function applyEnforceBounds({
...
-      skipProvingKey = true;
-    });
+  const descriptorBounds = pexToBounds(
+    proofRequest.request,
+    selectedCredentials,
+  );
+  let bounds = [];
+
+  descriptorBounds.forEach((items) => {
+    items.forEach((bound, idx) => {
```
Comment (cykoder) [inline packages/wasm/src/services/credential/bound-check.ts:80]: this is incorrect, the pextobounds result is split per credential. this wont work with multiple credentials.

descriptorBounds[0] corresponds to the first selected credential
descriptorBounds[1] corresponds to the second selected credential

**Review by cykoder — CHANGES_REQUESTED** [review]
needs changing for multiple credentials

File: packages/wasm/src/services/credential/bound-check.ts (hunk 1 above)
Comment (maycon-mello) [reply packages/wasm/src/services/credential/bound-check.ts:81]: yes, i've just pushed an update to support multiple credentials

Comment (maycon-mello) [reply packages/wasm/src/services/credential/bound-check.ts:81]: still finishing some tests

**Review by cykoder — DISMISSED** [review]

**Review by cykoder — APPROVED** [review]

**Pull request merged**